sdk/storage/azure-storage-file-share/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --v3 --python
 
 ### Settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.FileStorage/preview/2022-11-02/file.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.FileStorage/preview/2023-01-03/file.json
```

Comment:
next time can you update the swagger when you modify the generated code to be from a newer version? Before releasing autorest, we regenerate the storage libraries and run the checked in tests. If we generate from a different swagger than the generated code one uses, we can run into errors. thanks!